extensions/istio/client/src/main/java/io/fabric8/istio/client/V1alpha3APIGroupExtensionAdapter.java

```diff
@@ -19,15 +19,15 @@
 import io.fabric8.kubernetes.client.Client;
 import okhttp3.OkHttpClient;
 
-public class V1alpha3APIGroupExtensionAdapter extends APIGroupExtensionAdapter<V1alpha3APIGroupDSL> {
+public class V1alpha3APIGroupExtensionAdapter extends APIGroupExtensionAdapter<V1alpha3APIGroupClient> {
   @Override
   protected String getAPIGroupName() {
     return "v1beta1";
   }
```

Comment:
Is this string OK? should it be `v1alpha3`?

Reply:
It's not ok, thanks for spotting! Fixed.